templates/enabled_plugins.epp

```diff
@@ -0,0 +1,3 @@
+% This file managed by Puppet
+% Template Path: <%= $module_name %>/templates/enabled_plugins
+[<%= ($_plugins.uniq).join(',')%>].
```

Comment:
I think this needs `unique` vs `uniq` (puppet vs. ruby function)?

Even after updating the code in the calling scope and referencing `$::rabbimq::config::_plugins` (the private variable did end up working), there's some other stuff needed for this to work properly. Will try to throw up a diff soon.